TRANSLATION.md

```diff
@@ -1,76 +1,47 @@
-would like to contribute to the translation of nodejs.org, please refer to the following process:
+We use crowdin to manage translations. You can join the Node.js project on [Crowdin](https://crowdin.com/project/nodejs-website).
 
-> Since Apr, 1 2020 translation process moved to [Crowdin](https://crowdin.com/project/nodejs-website)
+## How to translate
 
-## Get started
+1. Join the [Crowdin](https://crowdin.com/project/nodejs-website)
+2. Select the language you want to translate
+3. Start translating
 
-1. Open [nodejs-website](https://crowdin.com/project/nodejs-website) Crowdin project
-2. Find your locale and start translation. Find more details in [guide for volunteer translators](https://support.crowdin.com/for-volunteer-translators/)
+## How to add a new language
 
-All translated and approved content will be pushed to this repo automatically. You don't need to create any PRs with translation. Just keep localization process on Crowdin.
+Go on `/i18n/config.json` and add the new language to the `locales` array.
 
-Original source can be found in [/locale/en](https://github.com/nodejs/nodejs.org/tree/main/locale/en). If you find any problem with original source, please create a PR with changes directly to `/locale/en`. Crowdin automatically pull all updates within 24 hours.
+Example of a language:
 
-### Can't find my locale on Crowdin
+```json
+{
+  "code": "fr",
+  "localName": "Français",
+  "name": "French",
+  "langDir": "ltr",
+  "dateFormat": "DD.MM.YYYY",
+  "hrefLang": "fr",
+  "enabled": true
+}
+```
 
-Please create a [new issue](https://github.com/nodejs/nodejs.org/issues/new?template=03-i18n.md) in this repo. Crowdin managers team would be happy to add new languages.
+- code: The language code. It must be the same as the folder name.
+- localName: The language name in its own language (it's use in language selector).
+- name: The language name in English.
+- langDir: The direction of the language. `ltr` for left to right, `rtl` for right to left.
+- dateFormat: The date format. It must be a valid [moment.js format](https://momentjs.com/docs/#/displaying/format/).
+- hrefLang: The language code in [ISO 639-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) format.
+- enabled: If the language is enabled or not.
 
-## Localization groups
+## How to become Proofreader
 
-An existing localization group is not required to start translation. You can contribute on Crowdin without it. Think about groups like a basement to communicate with other translators of your locale.
+- Have translated at least 1000 strings
+- Have 200 strings approved
 
-Contact your appropriate localization group, and discuss with them the best possible way to contribute. A list of the localization groups can be found below.
+To become a proofreader, you need to contact the [Node.js Crowdin Manager](https://crowdin.com/profile/ovflowd) and ask to be promoted. The manager will check your request and promote you if you are eligible.
```

Comment:
There are in total of 4 Crowdin Managers. Should we mention all of them?

Reply:
good question.
For Joe and Rich I don't they have time to manage Crowdin.

Reply:
I think we should make all of these information public, and we should not forget that: there is a cost of maintaining the correctness of these lines in the long term.

Reply:
Ok i will change that !